documents/validator.go

```diff
@@ -350,6 +373,7 @@ func PostAnchoredValidator(idService identity.ServiceDID, repo anchors.AnchorRep
 	return ValidatorGroup{
 		PreAnchorValidator(idService),
 		anchoredValidator(repo),
+		latestVersionValidator(repo),
```

Comment:
not sure adding the validator is a good idea. This is not re-usable if I want to use this for old documents. Create a new validator group and use that where you need.

Reply:
since this is called `PostAnchoredValidator, its should be ok no?

Reply:
I think is fine to have it inside the `PostAnchoredValidator`, `latestVersionValidator ` should be public, as I believe it will be invoked as well independently.